tuning/sft_trainer.py

```diff
@@ -322,6 +322,7 @@ def train(
     # this validation, we just drop the things that aren't part of the SFT Config and build one
     # from our object directly. In the future, we should consider renaming this class and / or
     # not adding things that are not directly used by the trainer instance to it.
+    train_args.use_cpu=True
```

Comment:
Please leave this change out of the PR, although useful note that transformers 4.42 requires this change to run unit tests on macs!

Reply:
Sure thanks!